templates/patterns/copyright_overlay/pattern-copyright-overlay.html.twig

```diff
@@ -0,0 +1,57 @@
+{#
+/**
+ * @file
+ * Copyright overlay pattern.
+ */
+#}
+{{ attach_library('oe_bootstrap_theme/copy_clipboard') }}
+
+{% set _trigger_label = trigger_label|default('View more'|t) %}
+
+{% set _classes = ['copyright-overlay', 'px-2', 'text-muted'] %}
+
+{% set _footer %}
+  {% include '@oe-bcl/button' with {
+    label: 'Copy Copyright'|t,
+    attributes:  create_attribute()
+      .setAttribute('data-copy-target', 'copyright-text'),
+  } only %}
+{% endset %}
+
+{% set _header %}
+  <h5 class="mb-0">{{ 'Copyright'|t }}</h5>
```

Comment:
for me the title can also be a field and default it to this

Reply:
added the title as a field too